systemtests/tests/virtualfull-bscan/testrunner

```diff
@@ -42,7 +42,7 @@ run job=$JobName level=Full yes
 wait
 messages
 list jobs
-@exec "sh -c 'touch ${tmp}/data/*.c'"
+@exec "sh -c 'touch ${tmp}/data/build/src/tests/*.c'"
```

Comment:
but that test previously ran flawlessly without this change - and now it is broken on my machine.
I don't understand why this was at all required.